src/content/docs/new-relic-solutions/new-relic-one/ui-data/maps-in-context.mdx

```diff
@@ -86,38 +110,49 @@ Here is more detail about the standard visual elements of a map:
   </tbody>
 </table>
 
-## Health status [#health]
+## See your health status at a glance, or use timewarp [#health]
+
+Some maps, our omnimap-powered maps included, use our standard [health status colors](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/view-entity-health-status-find-entities-without-alert-conditions/#colors) to indicate entity health. You can use the top pane to search for a specific entity within grouped entities, filter by health status or entity types, or adjust timewarp to explore health status over time.
 
-Some maps use our standard [health status colors](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/view-entity-health-status-find-entities-without-alert-conditions/#colors) to indicate entity health.
+<img 
+  title="Search, filter, and explore your entities"
+  alt="A screenshot of the top pane of the omnimap-powered service maps experience."
+  src={omnimapServiceMapsTopNav}
+/>
+
+Under the timewarp feature, you can explore the far right pane to see an activity stream, your service levels, reported data about metrics, deviations in behavior over time, data about your logs, and view your **Related entities**. Clicking **See full map** in **Related entities** is another method to access automap.
+
+<img 
+  title="Omnimap-powered service maps right pane"
+  alt="A gif that scrolls through the right pane in service maps. It shows different graphs that correspond to the above paragraph."
+  src={omnimapServiceMapsRightPane}
+/>
 
-Distributed trace maps use these colors to indicate health:
+As a note, distributed tracing maps use these colors to indicate health:
 
 * A solid red hexagon means one or more span errors.
 * A solid orange hexagon means one or more anomalous spans.
 
-## Map tooltip [#tool-tips]
+## Get golden metrics in the maps tooltip [#tool-tips]
 
-Maps in context show additional information about map elements when you hover over them.
+Our omnimap-powered tooltips show a snapshot of an entity's [golden metrics](/docs/apis/nerdgraph/examples/golden-metrics-entities-nerdgraph-api-tutorial). When you select **Map from this entity**, service maps will open up [automap](/docs/new-relic-solutions/new-relic-one/ui-data/automaps). 
```

Comment:
@akristen how can I tell whether I'm looking at an omnimap-powered map?

The top of this guide seems to say that it's offering an explanation of the omnimap-powered maps, but we discuss other types of maps, such as distributed tracing and external services. That's a bit confusing to me.